cloudinary-image-management-and-manipulation-in-the-cloud-cdn/php/class-connect.php

```diff
@@ -256,6 +256,15 @@ public function get_credentials() {
 		return $this->credentials;
 	}
 
+	/**
+	 * Get the cloud name if set.
+	 *
+	 * @return bool|mixed
+	 */
+	public function get_cloud_name() {
+		return $this->credentials['cloud_name'] ? $this->credentials['cloud_name'] : false;
```

Comment:
It would be cool if we returned `null` instead of `false` so we can avoid mixing the types here